src/auth/user.entity.ts

```diff
@@ -33,6 +36,9 @@ export class User {
   @Column({ default: false })
   isDeactivated: boolean;
 
+  @Column({ default: UserRole.USER })
```

Comment:
Because you did not define the **type** and **values**, the role is of the **varchar type**, which is wrong.
The Column should look like:
`@Column({ type: 'enum', enum: UserRole, default: UserRole.USER })`
Defining with type enum and values in the database **will appear as enum not as varchar**.
![user_role](https://user-images.githubusercontent.com/77193627/155894659-a0d138aa-2bb3-41f2-838c-ac033f35cc1e.PNG)